### ISSUE TITLE
3.10.3 - CTD before loading screen

### ISSUE THREAD

**Issue by Abros96** [LoneGazebo/Community-Patch-DLL#10198]
_1. Mod Version (X.Y.Z). Current Version: 3.10.3_
3.10.3

_2. DirectX Version_

_3. List of Other Mods_

_4. Describe the Issue_
When starting a new game CTD before the loading screen appears.
______________________________________________________________________________________
_5. Save Game From 1 Turn Before (ALWAYS ATTACH THIS IF POSSIBLE)_

_6. Logs (ALWAYS ATTACH THESE IF POSSIBLE)_

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/12641571/Logs.zip)

_7. CvMiniDump.dmp File (ATTACH IF REPORTING A GAME CRASH)_

[CvMiniDump.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/12641572/CvMiniDump.zip)

**Comment by Abros96** [comment]
Can I help with providing additional information, e.g. a list of enabled options in game setup?

**Comment by Abros96** [comment]
I can guarantee you that this is not an installation issue.

I verifed the integrity of the files, all ok. I can install and run the previous versions of VP without problems, everything is installed in the default orders, my only mod is VP with EUI.

Only after installing 3.10.x and clicking "Start game", when the screen with the leader portrait should appear, it crashes to desktop.

Don't the files I submitted show anything what goes wrong at that moment?

**Comment by jayeye2011** [comment]
Probably same bug. CTD VP 3.10.7 after Askia game setup (only domination victory enabled) start game then CTD before game map screen. Strange as this did not happen when trying a game setup with only science victory enabled (Korea).
[CvMiniDump.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/12659290/CvMiniDump.zip)
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/12659291/Logs.zip)

**Comment by axatin** [comment]
Yes, that's the same. It can happen whenever some victory condition is disabled, but it doesn't happen every time, and it also depends on some other factors, like which other civilizations are in the game.
